benchmarks/segmentation_layers.py

```diff
@@ -0,0 +1,296 @@
+import torch
+import nestedtensor
+import utils
+
+#
+# relu
+#
+def relu_tensor():
+    inputs = [
+            torch.randn(30, 500, 600),
+            torch.randn(30, 500, 600),
+            torch.randn(30, 500, 600),
+            torch.randn(30, 500, 600),
+            torch.randn(30, 500, 600),
+            torch.randn(30, 500, 600),
+        ]
+    
+    a = torch.stack(inputs)
+
+    def _relu_tensor():
+        torch.nn.functional.relu(a)
+    return _relu_tensor
+
+def relu_nt():
+    inputs = [
+            torch.randn(30, 500, 600),
```

Comment:
This sizes are "tensor compliant", so the benchmark should include something in its name that implies this. For now you're really only testing NestedTensor overhead with this.